web/source/keymanweb.js

```diff
@@ -3046,7 +3045,7 @@
             }
 
             //Display the loading delay bar (Note: only append 'keyboard' if not included in name.) 
-            var wText='Installing keyboard<br/>'+keymanweb._KeyboardStubs[Ln]['KN'].replace(/\s*keyboard\s*/i,'');
+            var wText='Installing keyboard<br/>' + kbdName;
```

Comment:
Nah, it was just original code I never bothered changing.  It's a related-enough refactor at this point, so I'll throw it in.